wiki/Gameplay/Grade/en.md

```diff
@@ -17,39 +26,33 @@ needs_cleanup: true
 | C | Over 60% 300s |
 | D | Anything else |
 
-A silver S or SS can be obtained by using the *[Hidden](/wiki/Game_modifier/Hidden)* and/or *[Flashlight](/wiki/Game_modifier/Flashlight)* mod.
-
 ## osu!taiko
 
 | Grade | Condition |
 | :-: | :-- |
-| SS | 100% Accuracy. |
-| S | Over 95% Accuracy (90+% GREATs, or less than 1 GOOD for every 10 notes). |
-| A | Over 90% Accuracy (80+% GREATs, or less than 1 GOOD for every 5 notes). |
-| B | Over 80% Accuracy (70+% GREATs, or less than 1 GOOD for every 3.33 notes). |
+| SS | 100% accuracy |
+| S | Over 95% accuracy (90+% GREATs, or less than 1 GOOD for every 10 notes) |
+| A | Over 90% accuracy (80+% GREATs, or less than 1 GOOD for every 5 notes) |
+| B | Over 80% accuracy (70+% GREATs, or less than 1 GOOD for every 3.33 notes) |
 
 ## osu!catch
 
 | Grade | Condition |
 | :-: | :-- |
-| SS | 100.00% Accuracy |
-| S | 98.01% to 99.99% Accuracy (an 'S' rank is possible even with several misses, like in *osu!mania*). |
-| A | 94.01% to 98.00% Accuracy |
-| B | 90.01% to 94.00% Accuracy |
-| C | 85.01% to 90.00% Accuracy |
-| D | Any other accuracy under 85.00%. |
-
-A silver S or SS can be obtained by using the *[Hidden](/wiki/Game_modifier/Hidden)* and/or *[Flashlight](/wiki/Game_modifier/Flashlight)* mod.
+| SS | 100% accuracy |
+| S | 98.01% to 99.99% accuracy (an S rank is possible even with several misses, like in osu!mania) |
+| A | 94.01% to 98.00% accuracy |
+| B | 90.01% to 94.00% accuracy |
+| C | 85.01% to 90.00% accuracy |
+| D | Any other accuracy under 85.00% |
 
 ## osu!mania
 
 | Grade | Condition |
 | :-: | :-- |
-| SS | 100% Accuracy (only MAX and/or 300). |
-| S | Over 95% Accuracy (an 'S' rank is possible even with several misses, like in *osu!catch*). |
-| A | Over 90% Accuracy. |
-| B | Over 80% Accuracy. |
-| C | Over 70% Accuracy. |
-| D | Anything else. |
-
-It is possible to obtain SSH or SH ranks (silver S or SS) with [Hidden](/wiki/Game_modifier/Hidden)/[Fade In](/wiki/Game_modifier/Fade_In) or [Flashlight](/wiki/Game_modifier/Flashlight) mod.
+| SS | 100% accuracy (only MAX and/or 300) |
```

Comment:
I'd probably drop this mention of "and/or 300":

```suggestion
| SS | 100% accuracy |
```

The reason for that is that the rank is directly correlated with the accuracy, and as I [recently brought up](https://github.com/ppy/osu-wiki/pull/6625), mania's accuracy calculation depends on ScoreV2 being active, and as such, with ScoreV1 it is sufficient to get either gold or rainbow 300s for an SS, while with ScoreV2 it needs to be exclusively rainbow 300s.